library/nats-streaming

```diff
@@ -1,6 +1,25 @@
 Maintainers: Ivan Kozlovic <ivan.kozlovic@apcera.com> (@kozlovic)
-
-Tags: 0.5.0, latest
 GitRepo: https://github.com/nats-io/nats-streaming-docker.git
-GitCommit: 3301b7b0e11bfd128eb4ba572fc1fdbfaab8d3e3
+GitCommit: 2f9abaf0830c64e40d22e3a8d4da47b96c666aef
+
+Tags: 0.5.0-linux, linux
+SharedTags: 0.5.0, latest
+amd64-GitCommit: 2f9abaf0830c64e40d22e3a8d4da47b96c666aef
```

Comment:
This section needs to include `Architectures: amd64, arm32v7, arm64v8` or it will inherit the default value, which is just `amd64`.

Reply:
Thanks, and for some reason I got it wrong with the release version! Let me update this. Sorry for wasting your time.